packages/testing/src/playwright/unstable/page-objects/organizationSwitcher.ts

```diff
@@ -18,8 +18,14 @@ export const createOrganizationSwitcherComponentPageObject = (testArgs: { page:
     expectNoOrganizationSelected: () => {
       return expect(page.getByText(/No organization selected/i)).toBeVisible();
     },
+    /**
+     * @deprecated Use `expectPersonalWorkspace` instead
+     */
     expectPersonalAccount: () => {
-      return expect(page.getByText(/personal account/i)).toBeVisible();
+      return expect(page.getByText(/personal (workspace|account)/i)).toBeVisible();
```

Comment:
Targeting both to avoid breaking changes if clerk-js is not released first or if the app has a older version.